sim/druid/talents.go

```diff
@@ -35,20 +35,13 @@ func (druid *Druid) BearArmorMultiplier() float64 {
 
 func (druid *Druid) ApplyTalents() {
 	druid.MultiplyStat(stats.Mana, 1.0+0.05*float64(druid.Talents.Furor))
-	// druid.AddStat(stats.SpellHit, float64(druid.Talents.BalanceOfPower)*2*core.SpellHitRatingPerHitChance)
 	// druid.AddStat(stats.SpellCrit, float64(druid.Talents.NaturalPerfection)*1*core.CritRatingPerCritChance)
 	// druid.PseudoStats.CastSpeedMultiplier *= 1 + (float64(druid.Talents.CelestialFocus) * 0.01)
-	// druid.PseudoStats.DamageDealtMultiplier *= 1 + (float64(druid.Talents.EarthAndMoon) * 0.02)
-	// druid.PseudoStats.SpiritRegenRateCasting = float64(druid.Talents.Intensity) * (0.5 / 3)
+	druid.PseudoStats.DamageDealtMultiplier *= 1 + (core.TernaryFloat64(druid.Talents.EarthAndMoon, 0.02, 0))
```

Comment:
This should be a spell mod with druid class mask as the talent has a class mask, thus not affecting spell damage procs like volcane or the weapon enchant and so on.

Reply:
I believe I changed this one before I had learned about SpellMods, I will move it there ofc :)

Reply:
I changed this to a spellmod with SpellMod_DamageDone_Pct, as according to druid discord it is multiplicative.

Also got rid of Intensity, Celestial Focus and Natural Perfection, the last of which is reworked and no longer has a passive.